client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -54,6 +59,30 @@ const PlanComponent = ( {
 	)
 		? JetpackBundleCard
 		: JetpackPlanCard;
+
+	if ( plan.upgradeable ) {
+		// TODO: remove this once we can purchase new plans.
+		// We need this now to make the nudge appear inside the Jetpack Security Bundle card
+		// and make it work as it was Jetpack Security Daily (monthly or annually).
+		const productSlug =
+			plan.productSlug === 'jetpack_security'
+				? 'jetpack_security_daily'
+				: 'jetpack_security_daily_monthly';
```

Comment:
Let's not hard-code strings in here. There are already constants defined for all of these strings.

Reply:
Sure, but let's keep in mind this is for testing purposes. That code should disappear in a couple of days (hopefully) when the backend is ready to support purchases.